docs/en/connector/source/Excel.md

```diff
@@ -0,0 +1,49 @@
+# Excel
+
+## Description
+
+Read data from excel file
+
+:::tip
+
+Engine Supported and plugin name
+
+* [x] Spark: Excel
+* [ ] Flink:
+
+:::
+
+## Options
+
+| name              | type    | required | default value |
+| ----------------- | ------- | -------- | ------------- |
+| path              | string  | yes      |               |
+| options.useHeader | boolean | yes      |               |
+| options.*         |         |          |               |
+
+### path[string]
+
+file path, starting with file://
+
+### options.useHeader[boolean]
+
+Whether to use the first row of Excel as the header
+
+### options.*
+
+For details, refer to the parameters of the third-party library [spark-excel](https://github.com/crealytics/spark-excel).
```

Comment:
After read this document, I can't know how to set options.*, I think move there option to here is better.

Reply:
> Can this plugin support custom header( or custom fields) in config now?

1、You can set whether to use the first line as the header, but it cannot be customized
2、You can set the area range for reading data through "DataAddress"